File: app/src/main/java/de/fhkiel/temi/robogguide/database/Items.java
package de.fhkiel.temi.robogguide.database;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;


@DatabaseTable(tableName = "items")
public class Items {
    @DatabaseField(generatedId = true)
    private int id;

    @DatabaseField
    private String name;

    @DatabaseField
    private int locations_id;

    public Items(){

    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getLocations_id() {
        return locations_id;
    }

    public void setLocations_id(int locations_id) {
        this.locations_id = locations_id;
    }
}

File: app/src/main/java/de/fhkiel/temi/robogguide/database/Items.kt
package de.fhkiel.temi.robogguide.database

import com.j256.ormlite.field.DatabaseField
import com.j256.ormlite.table.DatabaseTable

/**
 * Orm Data Class to to get the items table
 *
 * @property id
 * @property name
 * @property locations_id
 */

@DatabaseTable(tableName = "items")
data class Items(
    @DatabaseField(generatedId = true, columnName = "id")
    val id: Int,

    @DatabaseField(columnName = "name")
    val name: String,

    @DatabaseField(columnName = "locations_id")
    val locations_id: Int
) {

    constructor(): this(0,"",0)
}